src/Contract/Address.purs

```diff
@@ -118,6 +125,7 @@ import Ctl.Internal.Types.UnbalancedTransaction
 import Data.Array (head)
 import Data.Maybe (Maybe)
 import Data.Traversable (for, traverse)
+import Test.Ctl.Utils (assertTrue)
```

Comment:
We can't use this module. It is from the test dir, and using it breaks upstream, because `test/` dir is not included as a source dir when CTL is used as dependency